packages/web-components/src/components/image/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import Image from '../Image';
+
+# Image
+
```

Comment:
Hey @andysherman2121 , great PR! Want to suggest adding a short description of what this is, which is essentially a responsive image. May also want to describe what image items are and how to configure each with its own breakpoint. The example you have below doesn't have that at all so might not be as much value there, perhaps have an example with the media queries configured.

Reply:
Also perhaps an example with the description text as well.

Reply:
updated